kernel-rs/src/sysfile.rs

```diff
@@ -296,10 +297,10 @@ impl Kernel {
         let mut path: [u8; MAXPATH] = [0; MAXPATH];
         let p: *mut Proc = myproc();
         let mut data = &mut *(*p).data.get();
-        let _tx = self.fs().begin_transaction();
+        let tx = self.fs().begin_transaction();
         let path = ok_or!(argstr(0, &mut path), return usize::MAX);
```

Comment:
```suggestion
        let path = ok_or!(argstr(0, &mut path), return usize::MAX);
        let tx = self.fs().begin_transaction();
```

`sys_mknod()`에서 `begin_transaction()`의 위치를 설정한 의도와 맞추기 위한 수정입니다